docs/content/concepts/sui-move-concepts/packages/upgrade.mdx

```diff
@@ -377,7 +377,22 @@ The result includes an **Object changes** section with two pieces of information
 
 You can identify the different objects using the `Object.objectType` value in the response. The `UpgradeCap` entry has a value of `String("0x2::package::UpgradeCap")` and the `objectType` for the package reads `String("<PACKAGE-ID>::sui_package::<MODULE-NAME>")`
 
-To make sure your other packages can use this package as a dependency, you must update the manifest (Move.toml file) for your package to include this information.
+To make sure your other packages can use this package as a dependency, you must update the `Move.toml` manifest file for your package to include this information.
+
+<Tabs>
+
+<TabItem value="automated-address-management" label="Automated Addresses">
+
+Add an entry for the package in the `[addresses]` section that points to the value of the on-chain ID:
+
+```toml
+[addresses]
+sui_package = "<ORIGINAL-PACKAGE-ID>"
+```
```

Comment:
As mentioned, this should soon be unnecessary, just reflecting accuracy right now.